changelogs/unreleased/gh-6386-fix-table-equals-for-booleans.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/lua
+
+* Fix ``table.equals`` result when booleans compared (gh-6386).
```

Comment:
Nit: use the past tense here. See https://github.com/tarantool/doc/pull/2661/files.

Reply:
Fixed. Thanks.